tests/playbooks/kubevirt_pvc_img_upload.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Create a PVC from local VM image
+  hosts: localhost
+  connection: local
+  tasks:
+    - name: Create a PVC annotated for upload
+      # This will be shorter with kubevirt_pvc
+      k8s:
+       state: present
+       namespace: default
+       name: pvc-vm1
+       inline:
+         apiVersion: v1
+         kind: PersistentVolumeClaim
+         metadata:
+           name: pvc-vm1
+           labels:
+             app: containerized-data-importer
+           annotations:
+             cdi.kubevirt.io/storage.upload.target: ""
+         spec:
+           accessModes:
+           - ReadWriteOnce
+           resources:
+             requests:
+               storage: 20Mi
+
+    - name: Wait for PVC to become ready 
+      # Won't be needed with kubevirt_pvc with wait support
+      k8s_facts:
+        namespace: default
+        name: pvc-vm1
+        kind: PersistentVolumeClaim
+        api_version: v1
+      register: pvc
+      until: "pvc.resources[0].status.phase == 'Bound'"
+      delay: 10
+      retries: 30
+
+    - name: Upload local image to pvc
+      kubevirt_cdi_upload:
+        pvc_namespace: default
+        pvc_name: pvc-vm1
+        upload_host: https://127.0.0.1:55555
```

Comment:
Would be good to add documentation about how to set it up so people using (verifying) it would know how to do it.

Reply:
I was planning on expanding the CDI doc on this and then linking to that. Also there's a sneaky way for the module to circumvent the requirement for the cdi-uploadproxy to have a working Service+Ingress combo (which is what would end up as the `upload_host`) configured and instead do some kubectl port-forward magic.

Once I have the base functionality in place, I'll write a short description of that, so we can figure out whether we want to spend time on adding it.